Description: workaround @types/node update
Author: Yadd <yadd@debian.org>
Bug-Debian: https://bugs.debian.org/1007855
Forwarded: no
Last-Update: 2022-03-18

--- a/src/node.ts
+++ b/src/node.ts
@@ -47,6 +47,7 @@
   }
 
   getString(encoding = 'utf8'): string {
+// @ts-ignore
     return this.getBuffer().toString(encoding);
   }
 
--- a/src/volume.ts
+++ b/src/volume.ts
@@ -443,7 +443,9 @@
 }
 
 export function dataToStr(data: TData, encoding: string = ENCODING_UTF8): string {
+// @ts-ignore
   if (Buffer.isBuffer(data)) return data.toString(encoding);
+// @ts-ignore
   else if (data instanceof Uint8Array) return bufferFrom(data).toString(encoding);
   else return String(data);
 }
